File: Basic Problem Solving/Patterns/characterPatterns.py
ABCD
ABCD
ABCD
ABCD



n = int(input())

i = 1
while (i <= n):
    j = 1
    while(j <= n):
        character = chr(ord("A") + j - 1)
        print(character, end='')
        j += 1
    print()
    i += 1
    
    

-----------------

ABCD
BCDE
CDEF
DEFG


n = int(input())

i = 1
while (i <= n):
    j = 1
    start = chr(ord("A") + i - 1)
    while(j <= n):
        character = chr(ord(start) + j - 1)
        print(character, end='')
        j += 1
    print()
    i += 1

    
-----------------

A
BB
CCC
DDDD


n = int(input())

i = 1
while (i <= n):
    j = 1
    while (j <= i):
        char = chr(ord('A') + i - 1)
        print(char, end='')
        j += 1
    print()
    i += 1

File: Basic Problem Solving/Patterns/characterPatterns.cpp
ABCD
ABCD
ABCD
ABCD



#include <bits/stdc++.h>
using namespace std;

int main(){
    int n,i,j;
    cin >> n;
    i = 1;
    while (i <= n){
        j = 1;
        while (j <= n){
            char ch = 'A' + j - 1;
            cout << ch;
            j += 1;
        }
        cout << endl;
        i += 1;
    }
}
    

-----------------

ABCD
BCDE
CDEF
DEFG


#include <bits/stdc++.h>
using namespace std;

int main(){
    int n,i,j;
    cin >> n;
    i = 1;
    while (i <= n){
        j = 1;
        char start = 'A' + i - 1;
        while (j <= n){
            char ch =  start + j - 1;
            cout << ch;
            j += 1;
        }
        cout << endl;
        i += 1;
    }
}

    
-----------------

A
BB
CCC
DDDD

#include <bits/stdc++.h>
using namespace std;

int main(){
    int n,i,j;
    cin >> n;
    i = 1;
    while (i <= n){
        j = 1;
        char start = 'A' + i - 1;
        while (j <= i){
            cout << start;
            j += 1;
        }
        cout << endl;
        i += 1;
    }
}

          
